news_collector/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.sqlite3
+*.pyc
+__pycache__
+
```

Comment:
Don't add a separate .gitignore in this subfolder - use one at the root of the repository